test/e2e/windows/hybrid_network.go

```diff
@@ -79,6 +80,58 @@ var _ = SIGDescribe("Hybrid cluster network", func() {
 
 		})
 
+		ginkgo.It("should have stable networking for Linux and Windows with services", func() {
+			serviceName := "stable-networking"
+			ginkgo.By("creating service " + serviceName + " with type=ClusterIP in namespace " + f.Namespace.Name)
+			jig := e2eservice.NewTestJig(f.ClientSet, f.Namespace.Name, serviceName)
+
+			svc, err := jig.CreateTCPService(func(svc *v1.Service) {
+				svc.Spec.Type = v1.ServiceTypeClusterIP
+			})
+			framework.ExpectNoError(err)
+
+			ginkgo.By("creating some linux pods in the service")
+			var linuxPod *v1.Pod
+			for i := 0; i < 2; i++ {
+				ginkgo.By("creating a linux pod and waiting for it to be running")
+				linuxPod = createTestPod(f, linuxBusyBoxImage, linuxOS)
+				linuxPod.ObjectMeta.Labels = svc.Labels
+				linuxPod = f.PodClient().CreateSync(linuxPod)
+			}
+
+			ginkgo.By("creating some Windows pods in the service")
+			var windowsPod *v1.Pod
+			for i := 0; i < 2; i++ {
+				ginkgo.By("creating a windows pod and waiting for it to be running")
+				windowsPod = createTestPod(f, windowsBusyBoximage, windowsOS)
+				windowsPod.Spec.Containers[0].Args = []string{"test-webserver"}
+				windowsPod.ObjectMeta.Labels = svc.Labels
+				windowsPod = f.PodClient().CreateSync(windowsPod)
+			}
+
+			ginkgo.By("verifying Service connectivity from pods outside the service")
```

Comment:
for completeness, there is another dimension that seems not covered here, traffic from nodes to service, usually tested using  a hostnetwork pod, that was a source of bugs in the past

Reply:
This requires hostprocess container feature on windows, which is default in 1.23+ I've add this to the test if that feature is enabled in the test suite, which I've turned on by default.

Reply:
or just add it in a different test, if you put a lot of things in the same test you'll never be able to understand if they fail with the same root cause

Reply:
this is a great point, I can split this test into separate tests easily.